cli/src/test/java/com/devonfw/tools/ide/tool/tomcat/TomcatTest.java

```diff
@@ -16,36 +16,32 @@ public void testTomcat() {
     IdeTestContext context = newContext(PROJECT_TOMCAT);
     Tomcat tomcatCommandlet = new Tomcat(context);
 
-    // install
-    tomcatCommandlet.install();
-
     // run
-    tomcatCommandlet.command.setValue(TomcatCommand.START);
     tomcatCommandlet.run();
 
     // assert
     checkInstallation(context);
     checkDependencyInstallation(context);
-    checkRunningTomcat(context);
+    //    checkRunningTomcat(context);
   }
 
   private void checkDependencyInstallation(IdeTestContext context) {
 
-    assertThat(context).logAtInfo().hasEntries("The version 17.0.10_7 of the dependency java is being installed",
-        "The version 17.0.10_7 of the dependency java was successfully installed");
+    assertThat(context).logAtInfo().hasMessage("The version 17.0.10_7 of the dependency java is being installed");
+    assertThat(context).logAtInfo().hasMessage("The version 17.0.10_7 of the dependency java was successfully installed");
```

Comment:
Was this changed by accident? Before, I was checking also the order of the two log-statements. Why should that be wrong?

Reply:
This was a mistake by my side and is now changed back to hasEntries()

Reply:
Maybe on your local machine but not yet on github here in this PR.

Reply:
Some for `checkRunningTomcat` that was changed alike and then even commented out.

Reply:
Fixed.